pylint/checkers/stdlib.py

```diff
@@ -44,7 +45,7 @@
 from pylint.checkers import BaseChecker, DeprecatedMixin, utils
 from pylint.interfaces import IAstroidChecker
 
-OPEN_FILES = {"open", "file"}
+OPEN_FILES_MODE = {"open", "file"}
```

Comment:
```suggestion
OPEN_FILES_MODE = ("open", "file")
```
Why not use a tuple here? A set is a bit over the top for just two entries.

Reply:
I mainly did this because is was already as set. For example, on line 53 `ENV_GETTERS` is even a set with only one entry.
Want me to change both to a tuple? Or change this to a tuple and line 53 to a string?

Reply:
Let's stick with a tuple for `ENV_GETTERS`. That way the `if` statement doesn't need to change if we ever need to add additional entries.